woocommerce/woocommerce-android
# Issue/expand collapse grouped products

### Description
This PR adds collapse/expand behavior to grouped products. It does that by adding the new property isExpanded to Grouped products and check for that value to represent the state of the item (expanded/collapsed)

### Testing instructions
1. Open the orders list
2. Select an order containing bundled products or composite products
3. Check that now grouped products can be expanded/collapsed
4. Expand a grouped product and rotate the screen
5. Check that the expanded product maintains its state (it remains expanded)


### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/46d607f8-d890-420e-9510-7198fc7645a0
